File: market_mobile/lib/constants/order_rules.dart
/// Web [frontend/src/lib/orderRules.ts] ile aynı teslimat kuralları.
abstract class OrderRules {
  static const double freeShippingThresholdTl = 200;
  static const double standardShippingFeeTl = 29.9;

  static double shippingFeeTl(double subtotal) {
    return subtotal > freeShippingThresholdTl ? 0 : standardShippingFeeTl;
  }
}

File: frontend/src/lib/orderRules.ts
/** Sepet ve ödeme ekranlarında paylaşılan teslimat kuralları (tek kaynak). */
export const ORDER_RULES = {
  FREE_SHIPPING_THRESHOLD_TL: 200,
  STANDARD_SHIPPING_FEE_TL: 29.9,
} as const

export function shippingFeeTl(subtotal: number): number {
  return subtotal > ORDER_RULES.FREE_SHIPPING_THRESHOLD_TL
    ? 0
    : ORDER_RULES.STANDARD_SHIPPING_FEE_TL
}
